### PR TITLE
Add DeflectQml library

### PR DESCRIPTION
**Pull request by dnachbaur**

### CODE REVIEW

File: deflectQml/EventHandler.h [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+
+#include <QObject>
+#include <QSocketNotifier>
+
+#include <deflect/Stream.h>
+
+namespace deflect
+{
+
+class EventHandler : public QObject
```
Comment (rdumusc) [inline deflectQml/EventHandler.h]: rename EventHandler -> EventReceiver? handling is not done in here

File: deflectQml/EventHandler.h [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+{
+    Q_OBJECT
+
+public:
+    EventHandler( Stream& stream );
+    ~EventHandler();
+
+signals:
+    void pressed( double , double );
```
Comment (rdumusc) [inline deflectQml/EventHandler.h]: name variables to help potential users?

File: deflectQml/QmlStreamer.h [hunk 3]
```diff
@@ -0,0 +1,129 @@
...
+
+public:
+    /**
+     * Construct a new qml streamer by loading the QML, accessible by
+     * getRootItem() and sets up the Deflect stream.
+     *
+     * @param qmlFile URL to QML file to load
+     * @param streamName name of the Deflect stream
+     * @param streamHost hostname of the Deflect server
+     * @param size of the streamer
```
Comment (rdumusc) [inline deflectQml/QmlStreamer.h]: size -> can't be infered from QML file?

**Comment by dnachbaur** [comment]
Updated

**Comment by dnachbaur** [comment]
Ready to merge?

**Comment by rdumusc** [comment]
Ready to merge, perfect!

**Pull request merged**